redaxo/src/addons/be_style/plugins/redaxo/scss/_variables.scss

```
@@ -355,7 +355,7 @@ $dropdown-caret-color:           #000 !default;
 // Note: These variables are not generated into the Customizer.
 
 $zindex-navbar:            1000 !default;
-$zindex-dropdown:          1000 !default;
+$zindex-dropdown:          1065 !default;
```

```suggestion
$zindex-dropdown:          1065;
```

Die Datei ist ja eine Kopie der Original-Variables-Datei aus Bootstrap, und immer wenn wir Werte anpassen, nehmen wir das `!default` weg. So sieht man direkt, welche Werte noch die Originalwerte sind, und welche wir angepasst haben (gegenüber den BS-Defaults).